package.json

```diff
@@ -62,7 +64,8 @@
     "puppeteer-extra": "^3.3.6",
     "puppeteer-extra-plugin-stealth": "^2.11.2",
     "queue": "^7.0.0",
-    "systeminformation": "^5.22.7"
+    "systeminformation": "^5.22.7",
```

Comment:
These should probably go into `optionalDependencies` as they're build related, correct?

Reply:
That lib is used for getting the machine stats 🤔: https://github.com/browserless/browserless/blob/feat/debugger/src/monitoring.ts#L7-L23